### PR TITLE
[Auditbeat] fim(ebpf): enrich file events with container id

### PR DESCRIPTION
**Pull request by mmat11**
## Proposed commit message

fim(ebpf): enrich file events with container id

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ] Depends on https://github.com/elastic/beats/pull/38199

## Related issues

https://github.com/elastic/integrations/issues/7401

## Screenshot

image

### CODE REVIEW

**Comment by pkoutsovasilis** [comment]
run docs-build

File: auditbeat/module/file_integrity/event.go [hunk 1]
```diff
@@ -400,6 +401,10 @@ func buildMetricbeatEvent(e *Event, existedBefore bool) mb.Event {
 		out.MetricSetFields.Put("process", process)
 	}
 
+	if e.ContainerID != "" {
+		out.MetricSetFields.Put("container.id", e.ContainerID)
```
Comment (nicholasberlin) [inline auditbeat/module/file_integrity/event.go:405]: Should this be something like this:
```
container := mapstr.M{
    "id": e.ContainerID,
}

out.MetricSetFields.Put("container", container)
```

Or does `MetricSetFields.Put` handle dotted paths?

**Review by nicholasberlin — APPROVED** [review]
One question, but otherwise, LGTM

File: auditbeat/module/file_integrity/event.go (hunk 1 above)
Comment (pkoutsovasilis) [reply auditbeat/module/file_integrity/event.go:405]: so when you use `.Put(...)` the dot is separated automatically in a nested object as being done [here](https://github.com/elastic/beats/pull/38328/files#diff-908ccbec2a6448f00637da326cdfd893c01d540802728329ac3be463cd0585bbR419) but I can definitely see why it got your eye. If you want this changed no prob by me

Comment (nicholasberlin) [reply auditbeat/module/file_integrity/event.go:405]: TIL ... thanks LGTM

**Review by cmacknz — APPROVED** [review]

**Pull request merged**